docs/tasks/manage-stateful-set/upgrade-pet-set-to-stateful-set.md

```diff
@@ -0,0 +1,152 @@
+---
+assignees:
+- bprashanth
+- enisoc
+- erictune
+- foxish
+- janetkuo
+- kow3ns
+- smarterclayton
+
+---
+
+{% capture overview %}
+This page shows how to upgrade from Pet Sets (Kubernetes cluster at version 1.3-1.4) to Stateful Sets (version >= 1.5).
```

Comment:
Janet, We recently put together a [style guide](http://kubernetes.io/docs/contribute/style-guide/), and I realize now that I need to add some content to the style guide about API names. I've done that in [PR 1807](https://github.com/kubernetes/kubernetes.github.io/pull/1807). The idea is that we want to refer to API objects by using the exact API object name. So for example, we would say PetSet, PetSets, StatefulSet, StatefulSets, etc.